core/api/src/app/quiz/claim.ts

```diff
@@ -88,7 +101,18 @@ export const completeQuiz = async ({
 
     if (validatedIPMetadata instanceof UnauthorizedIPError) return validatedIPMetadata
 
-    return new UnknownRepositoryError("add quiz error")
+    return new UnknownRepositoryError("claim quiz error")
```

Comment:
didnt see this in the last PR... but `UnknownRepositoryError` should only be used in repositories. do we need a new error here?